packages/ag-charts-community/src/chart/series/polar/donutSeriesProperties.ts

```diff
@@ -90,6 +91,20 @@ class DonutSeriesCalloutLine extends BaseProperties {
 }
 
 export class DonutSeriesProperties extends SeriesProperties<AgDonutSeriesOptions> {
+    override isValid(): boolean {
+        if (!super.isValid()) {
+            return false;
+        } else if (this.innerRadiusRatio == null && this.innerRadiusOffset == null) {
```

Comment:
nit: you don't need `else` if the previous `if` `return` :)

Reply:
I tend to do it because it saves a line - but can change if that’s the consensus

Reply:
Hmmm... isn't the logic here wrong anyway, shouldn't we always perform the validations here?

Reply:
Updated